modules/components/pages/processors/aws_bedrock_embeddings.adoc

```diff
@@ -0,0 +1,164 @@
+= aws_bedrock_embeddings
+// tag::single-source[]
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates vector embeddings from text prompts, using the https://aws.amazon.com/bedrock/[AWS Bedrock API^].
+
```

Comment:
@rockwotj. Just this outstanding question or should I ask the PM?

Reply:
This note?

> When Redpanda Connect runs a data pipeline with a Ollama processor in it, Redpanda Cloud deploys a GPU-powered instance for the exclusive use of that pipeline. As pricing is based on resource consumption, this can have cost implications.

No it's unique to the Ollama one?